buf.yaml

```diff
@@ -415,7 +415,8 @@ lint:
     - external/habitat/event.proto # this file is owned by habitat and copied here
 breaking:
   use:
-    - PACKAGE
+    # TODO: TEMPORARY CHANGE JUST FOR THIS PR!
+    - FILE
```

Comment:
Need to allow the breaking change for this commit; will revert after this PR merges.

Reply:
Thanks! I appreciate that you're fixing these errors!